monitor/metrics/checker.go

```diff
@@ -75,7 +75,7 @@ func (mc *Checker) CheckPeers(peers []peer.ID) error {
 // and no alert has been sent before.
 func (mc *Checker) CheckAll() error {
 	for _, metric := range mc.metrics.AllMetrics() {
-		if mc.FailedMetric(metric.Name, metric.Peer) {
+		if metric.Name == "ping" && mc.Failed(metric.Peer) {
```

Comment:
We shouldn't be hardcoding this. We should alert on all failed metrics. It's not our business to decide whether alerts will be meaninful.